### ISSUE TITLE
Dataset problem: MCM-UA-1-0 has longitude values below 0

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#527]
**Describe the dataset issue**
The MCM-UA-1-0 model has longitude coordinates below 0. See for instance:
```
/badc/cmip6/data/CMIP6/CMIP/UA/MCM-UA-1-0/historical/r1i1p1f1/Omon/thetao/gn/latest/thetao_Omon_MCM-UA-1-0_historical_r1i1p1f1_gn_185001-201412.nc
```

```
cube.coord('longitude')
DimCoord(array([ -0.9375,   0.9375,   2.8125,   4.6875,   6.5625,   8.4375,
[...]
       347.8125, 349.6875, 351.5625, 353.4375, 355.3125, 357.1875]), bounds=array([[ -1.875,   0.   ],
       [  0.   ,   1.875],
       [  1.875,   3.75 ],
       [  3.75 ,   5.625],
       [  5.625,   7.5  ],
       [  7.5  ,   9.375],
[...]
       [354.375, 356.25 ],
       [356.25 , 358.125]]), standard_name='longitude', units=Unit('degrees'), long_name='longitude', var_name='longitude', circular=True)
```

This fails the CMOR checks:
```
esmvalcore.cmor.check.CMORCheckError: lon: has values < valid_min = 0.0

```

The dataset:
- Project CMIP6
- Full dataset decription (fill out as many as you know, please):
  - dataset: MCM-UA-1-0
  - experiment: historical and possibly others
  - mip: Omon
  - grid: gn
  - type:
  - version:
  - tier:
  - variable used:thetao and possibly others
  - data version:

Iris doesn't like longitude values below zero, or non-monotonic axes:

Sorry for all the dataset issue reports - almost every single CMIP6 model is failing me know. Almost nothing is CMOR compliant out the box.

**Comment by zklaus** [comment]
No need to apologize, @ledm! I think it's great to report these problems upstream, so that procedures can be improved.

**Comment by zklaus** [comment]
Also, it is not that iris doesn't like them (I had no problem reading the data with iris), but that it is forbidden by [CMIP6](https://docs.google.com/document/d/1os9rZ11U0ajY7F8FWtgU4B49KcB59aFlBVGfLC4ahXs/edit#bookmark=id.3t64x7z1je8t)(p.5 Requirements for coordinate variables, note 3).

**Comment by remi-kazeroni** [comment]
Any news on this? I cannot reproduce directly the issue with thetao because I get another issue with levels:
`esmvalcore.cmor.check.CMORCheckError: There were errors in variable thetao: 
Generic level coordinate olevel has wrong standard_name or is not set.`

But I get the issue of longitude below 0 with other variables like tos:
`esmvalcore.cmor.check.CMORCheckError: There were errors in variable tos:
lon: has values < valid_min = 0.0`

**Comment by remi-kazeroni** [comment]
To close this issue with the `thetao` variable, we need #1102 for the negative longitude and a fix à la #1098 for the wrong standard name of level coordinate.

**Comment by tomaslovato** [comment]
@remi-kazeroni I can try to pipeline this in #1098 (if you agree) ... after a quick look to the data it should be only necessary to roll the longitude value between 0..360

**Comment by remi-kazeroni** [comment]
> 
> 
> @remi-kazeroni I can try to pipeline this in #1098 (if you agree) ... after a quick look to the data it should be only necessary to roll the longitude value between 0..360

@tomaslovato yes, that would be great if you could solve both issues (negative longitude and level coordinate naming) in #1098 because I need to take care of quite a few fixes. This way I would close #1102. I'm happy to take a look at #1098 if needed.

**Comment by remi-kazeroni** [comment]
This can be closed now that #1098 is merged
